src/controllers/resource.ts

```diff
@@ -29,13 +29,29 @@ export class ResourceController {
 		check('alsoKnownAs.*.uri').isString().withMessage(Messages.Invalid),
 		check('alsoKnownAs.*.description').isString().withMessage(Messages.Invalid),
 	];
+	public static createResourceValidator = [
+		check('did').exists().isString().contains('did:cheqd').withMessage(Messages.InvalidDid),
+		check('jobId')
+			.custom((value, { req }) => {
+				if (!value && !(req.body.name && req.body.type && req.body.content)) return false;
+				return true;
+			})
+			.withMessage('name, type and content are required'),
+		check('name').optional().isString().withMessage(Messages.Invalid),
+		check('type').optional().isString().withMessage(Messages.Invalid),
+		check('content').exists().isString().withMessage(Messages.Invalid),
```

Comment:
content can be optional, it's not required when the user is submitting a signatureResponse. It is only required when jobId is not required. Which is already handled above